cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -225,24 +227,25 @@ protected void addService(String name, HttpServiceHandler handler, HttpServiceHa
    * @param schedule the schedule to be added for the Workflow
    * @param workflowName the name of the Workflow
    */
+  @Deprecated
```

Comment:
Just wonder, do we need to deprecate this? Will in future both `TimeSchedule` and `StreamSizeSchedule` both extends/implements `Schedule`? If it does, then this method is totally ok (in fact this method will be the method exposed through the configurer).